index.html

```diff
@@ -2778,9 +2789,11 @@ <h3>Terms of Use</h3>
       "name": "Bachelor of Science and Arts"
     }
     },
-    "proof": { <span class="comment">...</span> }
+    "proof": { 
+
+  }
   }],
-  <span class="highlight">"termsOfUse": [{
+  "termsOfUse": [{
```

Comment:
Why are we removing the highlight? We highlight the parts of examples to draw people's attention to the feature being focused on in the example.

Reply:
It was unintentional from quick copying and pasting from JSON-LD playground after confirming the document was valid. I didn't think about the fact I need to make it formatted back into HTML. Thanks for catching these things. I'll be more careful about keeping these things consistent going forward.